src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.AgentPool.ps1

```diff
@@ -229,6 +233,49 @@ class AgentPool: ADOSVTBase
         return $controlResult
     }
 
+    hidden [ControlResult] CheckAutoUpdateAutomatedFix([ControlResult] $controlResult)
+    {
+        try 
+        {
+            #Backup data object is not required in this scenario.
+            $RawDataObjForControlFix = @();
+            $RawDataObjForControlFix = ([ControlHelper]::ControlFixBackup | where-object {$_.ResourceId -eq $this.ResourceId}).DataObject
+
+            $body = ""
+
+            if (-not $this.UndoFix)
+            {                 
+                if ($body.length -gt 1) {$body += ","}
```

Comment:
?? if ($body.length -gt 1) this condition will never come true here, just above 2 line $body is declared as $body = ""